src/main/java/com/mongodb/spark/sql/connector/schema/ConverterHelper.java

```diff
@@ -31,6 +31,8 @@
 
 /** Shared converter helper methods and statics */
 public final class ConverterHelper {
+  static final SchemaToExpressionEncoderFunction SCHEMA_TO_EXPRESSION_ENCODER_FUNCTION =
```

Comment:
Make static the function that is used to convert a schema to an expression encoder.